docs/ethical-advertising.rst

```diff
@@ -135,6 +135,9 @@ Opting Out
 
 We have added multiple ways to opt out of the advertising on Read the Docs.
 
+Users can go ad-free for as long as they are logged-in
+by becoming a `Gold Member to Read the Docs <https://readthedocs.org/accounts/gold/>`_.
```

Comment:
For some reason the `to` feels weird here. I guess it implies a subscription? That makes sense, just reads weird to me and I'm not 100% sure why.

Reply:
I think `of` is better than `to`.